docusearch/tests/test_loading.py

```diff
@@ -0,0 +1,22 @@
+from django.test import TestCase
+
+from docusearch.management.commands import import_documents as importer
+
+
+class ImportTest(TestCase):
+
+    def test_is_date(self):
+        """ For our specfic use-case, a string contains a date if it's all
```

Comment:
s/specfic/specific and maybe change `Let's test that.` to `Test if function correctly identifies dates`